boa/contracts/vvm/vvm_contract.py

```diff
@@ -51,10 +67,26 @@ def deploy(self, *args, env=None):
         if env is None:
             env = Env.get_singleton()
 
-        address, _ = env.deploy_code(bytecode=self.bytecode + encoded_args)
+        address, _ = env.deploy_code(bytecode=self.bytecode + encoded_args, **kwargs)
 
         return self.at(address)
 
+    def deploy_as_blueprint(
+        self, env=None, blueprint_preamble=DEFAULT_BLUEPRINT_PREAMBLE, **kwargs
+    ):
+        """
+        Deploy a new blueprint from this contract.
+        :param blueprint_preamble: The preamble to use for the blueprint.
+        :param env: The environment to deploy the blueprint in.
+        :param kwargs: Keyword arguments to pass to the environment `deploy_code` method.
+        :returns: A contract instance.
+        """
+        return VVMDeployer(
```

Comment:
Why construct a new VVMDeployer? It could be expensive (maintenance wise, I wouldn't assume anything besides a "plain old data type" is cheap to construct).